docs/UserGuide.md

```diff
@@ -334,20 +342,35 @@ to add a meeting from 1330 to 1430 when you already have one scheduled for
 
 #### 5.2.2 List meetings: `listMeeting`
 
-Show the list of meetings in MyInsuRec.
-If used with the optional parameter `[d/DATE]`, *MyInsuRec* will show only meetings happening in that time period.
+Shows a list of meetings in MyInsuRec.
 
 Format: `listMeeting [d/DATE]`
 
+<div markdown="span" class="alert alert-info">:information_source: **Note:**
+* You can use the `d/DATE` parameter optionally to view the list of meetings happening in that time period!
 * `DATE` is specified by keywords. The possible keywords are:
-  * `tomorrow` for a list of meetings happening today or tomorrow;
-  * `week` for a list of meetings happening today or in the next 7 days;
-  * `month` for a list of meetings happening this month.
+  * `tomorrow` for a list of meetings happening tomorrow;
+  * `week` for a list of meetings happening in the next week;
+  * `month` for a list of meetings happening in the next month.
+* For example, `listMeeting d/week` will show a list of meetings happening in the next week.
+* This is an excellent feature if you want to get a quick overview of your upcoming schedule!
+</div>
 
-Examples:
-* `listMeeting`
-* `listMeeting d/tomorrow`
-* `listMeeting d/month`
+<div markdown="span" class="alert alert-success">:exclamation: **Tips and tricks:**
+This command is used to get the index of a meeting. In order to perform commands related to a particular meeting such as `editMeeting`, you will have to first get its index by running `listMeeting`. So, expect to use this command a lot!
+</div>
+
+Example inputs:
+1. `listMeeting`
+2. `listMeeting d/month`
+
+Expected behavior:
+1. Displays **a list of all the meetings**.
+2. Displays **a list of meetings happening in the next month**.
```

Comment:
Would suggest merging these component together
example

Display **a list of all the meetings**
1. listMeeting

Reply:
hmm im not sure i understand this, my intention was to show the two different inputs bc one has the date parameter.
do you mean to combine them like:

list all meeting
input: listmeeting
expected behavior: all meeting

list meeting by time period
...

Reply:
<img width="452" alt="image" src="https://user-images.githubusercontent.com/63795894/198702215-a1af8b19-9f26-410c-ae8b-fece7dc81a65.png">
something like this?

Reply:
yea sth like this, but for this example maybe without the expected behaviour portion.

so sth like 

`List all meetings`
&nbsp;   `-Example input:`
&nbsp;&nbsp;&nbsp;&nbsp;          `-listMeeting`